user_vfp dosen't exist on musl's sys/user.h either (actually pretty much on
no modern libcs).

--- third_party/crashpad/crashpad/util/linux/thread_info.h	2020-10-04 19:04:27.902955377 +1100
+++ third_party/crashpad/crashpad/util/linux/thread_info.h	2020-10-04 19:04:30.426969313 +1100
@@ -304,9 +304,6 @@
                 "Size mismatch");
 #elif defined(ARCH_CPU_ARMEL)
   static_assert(sizeof(f32_t::fpregs) == sizeof(user_fpregs), "Size mismatch");
-#if !defined(__GLIBC__)
-  static_assert(sizeof(f32_t::vfp) == sizeof(user_vfp), "Size mismatch");
-#endif
 #elif defined(ARCH_CPU_ARM64)
   static_assert(sizeof(f64) == sizeof(user_fpsimd_struct), "Size mismatch");
 #elif defined(ARCH_CPU_MIPS_FAMILY)
